### DISCUSSION TITLE
brave-browser cask will not upgrade to the latest version (1.32.133)

### DISCUSSION THREAD

**Discussion by jcraig999** [Homebrew/discussions#2543] [Casks] [answered]
I'm not quite sure how to (or if I should), start an issue in Homebrew/cask, but brave browser will not update to the latest version when installed via cask. I prefer to update all my applications via brew instead of manually inside the application. When I downloaded brave manually, it was already at the latest version. But when I download it via cask, it is still stuck at 1.32.106. Can anybody confirm that this is the case for them? Here are some screenshots to better illustrate the situation:
# Downloading brave from website
![Downloading brave from website](https://images2.imgbox.com/40/ff/rdBBBEss_o.png)
## manually downloaded brave version 
![This is an image](https://images2.imgbox.com/88/89/iJ6dXoLB_o.png)

# brave-browser cask
## installing cask
![This is an image](https://images2.imgbox.com/ed/7f/ozLRID4P_o.png)
### cask version 
![This is an image](https://images2.imgbox.com/a4/40/51LgzBv7_o.png)
#### cask about page
It forces you to re-launch the browser in order to update instead of having it updated already in brew cask.
![This is an image](https://images2.imgbox.com/2f/9a/uMeSVJC1_o.png)

**Comment by neutric** [comment, marked as answer]
https://github.com/Homebrew/homebrew-cask/pull/114954 seeks to update the cask definition from 1.32.106 to the current version 1.32.113. As this pull request is not merged into the master branch just yet, `brew install --cask brave-browser` will install the version that is [currently defined](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/brave-browser.rb#L5) in the cask, which is 1.32.106. So all is working as intended, although intended sometimes means outdated.
